EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,109 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7852
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.